docker-compose.yml

```diff
@@ -693,6 +693,29 @@ services:
         aliases:
           - edx.devstack.xqueue_consumer
 
+  enterprise-subsidy:
+    image: edxops/enterprise-subsidy-dev
+    container_name: edx.devstack.enterprise-subsidy
+    hostname: enterprise-subsidy.devstack.edx
+    depends_on:
+      - mysql80
+      - memcached
+    command: bash -c 'while true; do python /edx/app/enterprise-subsidy/manage.py runserver 0.0.0.0:18280; sleep 2; done'
+    stdin_open: true
+    tty: true
+    environment:
+      DB_HOST: edx.devstack.mysql80
+      DB_PORT: 3306
+      DB_USER: enterprise001
```

Comment:
You can name the user to `subsidy001` as their are multiple enterprise services.